webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -190,6 +191,14 @@ const PerformanceAnalysisTable: React.FC<Props> = ({
     selectedMetricPerFilterOption,
   ]);
 
+  const fieldWithTooltip = (tooltip: string, label: string) => (
+    <Tooltip title={tooltip}>
+      <Typography variant="caption" fontSize={14}>
```

Comment:
This is not a caption. Instead, just inherit the parent's style:
```suggestion
      <Typography variant="inherit">
```

Reply:
okay, yes inherit would do the same as we..I can change the variant as suggested. thanks